atlasdb-service/src/main/java/com/palantir/atlasdb/jackson/TableMetadataDeserializer.java

```diff
@@ -55,8 +60,35 @@ public TableMetadata deserialize(JsonParser jp, DeserializationContext ctxt) thr
         } else {
             col = deserializeNamedCols(node);
         }
+        // TODO: make defaults not duplicated with TableMetadata code.
+        ConflictHandler conflictHandler = Optional.ofNullable(node.get("conflictHandler"))
+                .map(JsonNode::textValue).map(ConflictHandler::valueOf).orElse(ConflictHandler.IGNORE_ALL);
```

Comment:
The default in TableMetadata is `RETRY_ON_WRITE_WRITE`, is this intentionally different?